clar2wasm/src/tools.rs

```diff
@@ -654,6 +712,37 @@ fn compare_events(events_a: &[EventBatch], events_b: &[EventBatch]) {
     }
 }
 
+pub enum Network {
+    Mainnet,
+    Testnet,
+}
+
+pub fn crosscheck_with_network(
```

Comment:
If you rebase the PR now there's a `crosseval` that can reduce some boilerplate in here (and probably remove the need for `interpret_at_with_network` and `evaluate_at_with_network`)

Reply:
Nice addition! Changed.